packages/block-editor/src/components/inserter-list-item/index.js

```diff
@@ -83,9 +100,23 @@ function InserterListItem( {
 						disabled={ item.isDisabled }
 						onClick={ ( event ) => {
 							event.preventDefault();
-							onSelect( item );
+							onSelect(
+								item,
+								isAppleOS() ? event.metaKey : event.ctrlKey
```

Comment:
Don't we have logic for this in the keyboard package?

Reply:
It doesn't seem exposed, but definitely something we should expose.

Reply:
mmm, actually, it seems I could use `isKeyboardEvent` directly. Thanks Ella

Reply:
wasn't able to do on the "click" event, we'd need to expose isApple somehow.